components/hyper_serde/lib.rs

```diff
@@ -604,15 +605,15 @@ impl<'de> Visitor<'de> for StatusVisitor {
     }
 }
 
-impl<'de> Deserialize<'de> for De<Tm> {
+impl<'de> Deserialize<'de> for De<SystemTime> {
```

Comment:
It looks like serde already supports serialization of `SystemTime`. Maybe removing this code and relying on the implementation from serde will fix the issue you are having with the tests?

https://github.com/serde-rs/serde/pull/949